doc/development_process/issues.rst

```diff
@@ -12,3 +12,20 @@ identifiers.
 
 At any time it should be possible to establish the origin of a change and the
 reason behind it by following the references in the code.
+
+Reporting a regression issue
+****************************
+
+It could happen that the issue being reported is identified as a regression,
+as the use case is known to be working on earlier commit or release.
+In this case, providing directly the guilty commit when submitting the bug
+gains a lot of time in the eventual bug fixing.
+
+To identify the commit causing the regression, several methods could be used,
+but tree bisecting method is an efficient one that doesn't require deep code
+expertise and can be used by every one.
+
+For this, `git bisect`_ is the recommended tool.
```

Comment:
Please add a note reminding people that they need to run `west update` on each bisection step.

Reply:
you really do that? :)
This has been working for me for a while, never have to worry about west update anymore.

https://github.com/zephyrproject-rtos/zephyr/discussions/28651

Reply:
Note added